sample-compose/data/build.gradle.kts

```diff
@@ -45,8 +45,10 @@ android {
     testOptions {
         unitTests.all {
             if (it.name != "testDebugUnitTest") {
-                it.extensions.configure(kotlinx.kover.api.KoverTaskExtension::class) {
-                    isDisabled.set(true)
+                kover {
+                    excludeTests {
+                        tasks(it.name)
+                    }
                 }
             }
         }
```

Comment:
same, [this approach is deprecated](https://github.com/Kotlin/kotlinx-kover/blob/b35990f1a922cb793977b9627f2a7586fa3533bc/README.md#configuring-jvm-test-task). Using `mergeWith("stagingDebug")` will trigger the `data:testDebugUnitTest`.